### PR TITLE
Minor/planet

### PR DESCRIPTION
**Pull request by chris34**
Some smaller changes which came up spontaneously.

### CODE REVIEW

File: inyoka_theme_default/templates/planet/index.html [hunk 1]
```diff
@@ -16,15 +16,20 @@
...
 {% block planet_content %}
+  {% set rendered_pagination = macros.render_pagination(pagination) %}
+
+  {{ rendered_pagination }}
   {% if days %}
     {% if page == 1 %}
       <div class="planet-newestarticles">
         <h2>{% trans %}Recent articles{% endtrans %}</h2>
-        <ul class="list-inline">
+        <ol class="planet-newestarticles-list">
```
Comment (Lyra2108) [inline inyoka_theme_default/templates/planet/index.html:26]: Why did you change from ul to ol?

File: inyoka_theme_default/static/style/inyoka/planet.less [hunk 2]
```diff
@@ -10,16 +10,10 @@
...
   &-newestarticles {
-    .list-inline {
-      text-align:justifed;
-
-      li {
-        padding: 0;
-
-        &:not(:last-child):after {
-          color: @gray-light;
-          content: ' |';
```
Comment (Lyra2108) [inline inyoka_theme_default/static/style/inyoka/planet.less:21]: Why did you remove the item separators?

File: inyoka_theme_default/templates/planet/index.html [hunk 3]
```diff
@@ -77,15 +82,24 @@
                     <div class="text">{{ article.text }}</div>
                   </div>
                 </div>
-                <div class="planet-article-footer">
-                  <a href="{{ article|url|e }}">{% trans %}Permanent link{% endtrans %}</a>
+
+                <ul class="planet-article-footer">
```
Comment (Lyra2108) [inline inyoka_theme_default/templates/planet/index.html:86]: Why the change from div to a list?

File: inyoka_theme_default/templates/planet/index.html (hunk 1 above)
Comment (chris34) [inline inyoka_theme_default/templates/planet/index.html:26]: if the recent articles are displayed in two columns, it would be not directly clear which article is the newest

File: inyoka_theme_default/static/style/inyoka/planet.less (hunk 2 above)
Comment (chris34) [inline inyoka_theme_default/static/style/inyoka/planet.less:21]: To only use bootstrap's `.list-inline`. Thus, dropping the old style from uu.de which is not very typographic btw.

File: inyoka_theme_default/templates/planet/index.html (hunk 3 above)
Comment (chris34) [inline inyoka_theme_default/templates/planet/index.html:86]: 1. to use `.list-inline`
2. it saves markup if the styles are directly applied to the `ul.planet-article-footer` instead of a div-wrapper
3. just look at the pure HTML in w3m etc.: Which markup tells more? (rhetoric question…)

File: inyoka_theme_default/templates/planet/index.html (hunk 1 above)
Comment (Lyra2108) [inline inyoka_theme_default/templates/planet/index.html:26]: But they weren't placed in colums oO they could have been in two rows but than the first was always the first in the first row

File: inyoka_theme_default/static/style/inyoka/planet.less (hunk 2 above)
Comment (Lyra2108) [inline inyoka_theme_default/static/style/inyoka/planet.less:21]: but if I see it correct that's only a display:inline-block. and i prefer the " | " as a speration of the links.

File: inyoka_theme_default/templates/planet/index.html (hunk 1 above)
Comment (chris34) [inline inyoka_theme_default/templates/planet/index.html:26]: well, https://github.com/chris34/theme-default/commit/e655d01dbb3d4f8ed721908e9fa404cd406bab8f should cause this, so merge the autoprefixer-PR locally or checkout the commit before. And dont wonder if no numbers appear in Chrom{e,ium} neither after that, see the commit-mesage of https://github.com/chris34/theme-default/commit/a0e0cf6c22161e365a1b1e23ed18225cb3dcdae2 :D

File: inyoka_theme_default/templates/planet/index.html (hunk 3 above)
Comment (Lyra2108) [inline inyoka_theme_default/templates/planet/index.html:86]: 1. 
2. no it doesn't because you only replace the div with the ul and add an `<li>` for each element. So actually you have more markup.
3. It depends on your point of view. ;)

Comment (chris34) [inline inyoka_theme_default/templates/planet/index.html:86]: 1. if i rely on 1. (as a bonus consequence of 3.) i removed the wrapper-div and it is one element less. Or should i remove the `</li>`-tags to sparse even more markup? :P

File: inyoka_theme_default/static/style/inyoka/planet.less (hunk 2 above)
Comment (chris34) [inline inyoka_theme_default/static/style/inyoka/planet.less:21]: yeah,  `.list-inline` is basically a inline-block with padding. However, on mobile devices the separation via '| ' will get interesting to style – especially if the list fills multiple lines.

File: inyoka_theme_default/templates/planet/index.html (hunk 1 above)
Comment (chris34) [inline inyoka_theme_default/templates/planet/index.html:26]: as i merged #117, i rebased and squashed the two column-commits, so it should now work without further work

**Comment by Lyra2108** [comment]
If the list-style is fixed, this pr can be merged
